float32 pos_0 
float32 pos_1 
float32 max_velo 
float32 accel
float32 dt
---
float32[] ramp

